source/basic.tex

```diff
@@ -1336,13 +1332,11 @@
 \end{example}
 
 \pnum
-A name used in the definition of a class \tcode{X} outside of a member
-function body, default argument, \grammarterm{noexcept-specifier},
-default member initializer\iref{class.mem},
-contract condition\iref{dcl.attr.contract},
-or nested class definition\footnote{This refers to unqualified names
-following the class name; such a name may be used in the
-\grammarterm{base-clause} or may be used in the class definition.}
+A name used in the definition of a class \tcode{X} outside of a
+complete-class context\iref{class.mem} of \tcode{X}\footnote{This
+refers to unqualified names following the class name;
+such a name may be used in the \grammarterm{base-clause} or
+may be used in the class definition.}
```

Comment:
This (pre-existing) footnote text is appalling :(

"A name used in the definition of a class [...] refers to [...] a name used in the *base-clause* or [...] in the class definition".

What? So "in the definition of a class" is a broader syntactic category than "in the class definition"? I'll fix that after merging.

Reply:
2fdaf88f